src/misc.ts

```diff
@@ -189,6 +197,16 @@ export const transformOrderStatus = (
         localized: intl.formatMessage(orderStatusMessages.unconfirmed),
         status: StatusType.NEUTRAL
       };
+    case OrderStatus.PARTIALLY_RETURNED:
+      return {
+        localized: intl.formatMessage(orderStatusMessages.partiallyReturned),
+        status: StatusType.NEUTRAL
+      };
+    case OrderStatus.RETURNED:
+      return {
+        localized: intl.formatMessage(orderStatusMessages.returned),
+        status: StatusType.ERROR
```

Comment:
Should the status be `ERROR` for a RETURNED order status?

Reply:
Can't find it on the designs. 
@pwgryglak which dot color should we display on the order list if order is refunded / partially refunded?

Reply:
I talked with @pwgryglak and he mentioned the tag should be informational, so I believe `StatusType.NEUTRAL` should do the trick.